railties/lib/rails/generators/rails/app/templates/Dockerfile.tt

```diff
@@ -23,9 +23,10 @@ RUN apt-get update -qq && \
 # Install JavaScript dependencies
 ARG NODE_VERSION=<%= dockerfile_node_version %>
 ARG YARN_VERSION=<%= dockerfile_yarn_version %>
-ENV VOLTA_HOME="/usr/local"
-RUN curl https://get.volta.sh | bash && \
-    volta install node@$NODE_VERSION yarn@$YARN_VERSION
+RUN curl -fsSL https://fnm.vercel.app/install | bash && \
+    /root/.local/share/fnm/fnm install $NODE_VERSION
```

Comment:
I wonder if we really need a version manager for this. I looked at the implementation and it basically just download a tarball from https://nodejs.org/dist/ and extract it.

I don't really see a reason not to do that ourselves and skip a middleman.